vignettes/tcplfit2-vignette.Rmd

```diff
@@ -362,3 +362,148 @@ The output table resulting from the previous code chunk is the same format as th
     concRespPlot(res_wide[i,],ymin=-50,ymax=50)
   }
 ```
+
+## Example 5: How to "Censor" BMD
+The estimated BMD can occurs above or below an experimental dose or other specified values; e.g., it is higher or lower than the maximum or minimum concentration tested in the data. When this happens, `tcplhit2_core` and `concRespCore` provide options for users to "censor" BMD with use of arguments `bmd_low_bnd` and `bmd_up_bnd`. This section will provide examples to demonstrate how to do that. 
+
+`bmd_low_bnd` and `bmd_up_bnd` are multipliers applied to the minimum or maximum concentrations in the data to generate lower or upper bound for BMD estimates. 
+
+First, let's consider a situation when BMD is below the lowest concentration tested. This situation could happen when the experiment was not designed to have enough low dose groups to capture behavior in low dose region and it can be a red flag for unreliable analysis. The following code chunk loads data of a sample from `mc3` and simulates an example like this. Data from the first couple low dose groups were removed so that the responses in the first couple groups left would increase quickly. 
+```{r example 5 lower}
+# load example data
+spid <- unique(mc3$spid)[94]
+temp <- mc3[is.element(mc3$spid,spid),]
+
+# The data file has stored concentration in log10 form, fix it 
+conc <- 10**temp$logc # back-transforming concentrations on log10 scale
+resp <- temp$resp
+
+# modify the data for demonstration purposes 
+conc2 <- conc[conc>0.41]
+resp2 <- resp[which(conc>0.41)]
+
+# pull out all of the chemical identifiers and the name of the assay
+dtxsid <- temp[1,"dtxsid"]
+casrn <- temp[1,"casrn"]
+name <- temp[1,"name"]
+assay <- temp[1,"assay"]
+
+# create the row object
+row <- list(conc = conc2, resp = resp2, bmed = 0, cutoff = cutoff, onesd = onesd,
```

Comment:
Need to make sure this example can be run "as standalone code".  Though the Rmd knits if users do not know the cutoff for this example then this code will not work.  Make sure that all the necessary inputs as part of "row" are stated in the code for this example.